doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -205,18 +205,15 @@ In Keystone v1 and v2 (both deprecated), the term "tenant" was used in OpenStack
 
 <a name="3.3.3"></a>
-### 3.3.3. Host aggregates providing resource pooling
-Availability zones: provide resiliency and fault tolerance for VM deployments, by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms
+### 3.3.3. Cloud partitioning: Host Aggregates, Availability Zones
+Cloud administrators can partition the hosts within an OpenStack cloud using Host Aggregates and Availability Zones.
 
-Host aggregate: is a Cloud Admin concept which is used to map the VNFC instances on the compute nodes
+A Host Aggregate is a group of hosts (compute nodes) with specific characteristics and with the same specifications, software and/or hardware properties. Example would be a Host Aggregate created for specific hardware or performance characteristics. The administrator assigns key-value pairs to groups of compute nodes, this information will be used to schedule VMs. A host can belong to multiple host aggregates. Host Aggregates are not explicitly exposed to tenants.
 
-A host aggregate is a set of hosts with specific properties (multiple software and/or hardware properties); the properties are specified as key-value pairs. Example would be a host aggregate created for a particular flavour or specific hardware. A host can belong to multiple host aggregates. Host aggregates are not visible to users.
+Availability Zones (AZs) rely on Host Aggregates and make the partitioning visible to tenants. They are defined by attaching specific metadata information to an aggregate, making the aggregate visible for tenants. Hosts can only be in a single Availability Zone. By default a host is part of a default Availability Zone, even if it doesn’t belong to an aggregate. Availability Zones can be used to provide resiliency and fault tolerance for workloads deployments, for example by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms. They permit rolling upgrades – an AZ at a time upgrade with enough time between AZ upgrades to allow recovery of tenant workloads on the upgraded AZ. Finally, AZs can accommodate hosts with special hardware and software characteristics, for example, hosts with hardware accelerators.
```

Comment:
When would one use AZs for special hardware and software characteristics, compared to the host aggregate and metadata approach above?

Reply:
@kedmison W.r.t. "When would one use AZs for special hardware and software characteristics, compared to the host aggregate and metadata approach above?" when we want to expose to tenants; as you know tenant do not know anything about host-aggregates.

Host aggregates are grouping of resources that have some common characteristic; the resource can belong to multiple host aggregates. Host aggregates do not partition resources in the same fashion as an AZ.

We can create host aggregates that correspond to instance profiles (viz., flavor geometry and capabilities). But we can create an AZ with hosts configured to support many instance profiles.

W.r.t. AZs able to accommodate hosts with special hardware and software (while not necessary) but can be done where we need to earmark these special resources, potentially expensive, to certain tenants.

Reply:
This sentence "Finally, AZs can accommodate hosts with special hardware and software characteristics, for example, hosts with hardware accelerators." seems to be very confusing since that is pretty much what host aggregates and NFVI profiles/Openstack flavours are for: identifying special hosts and scheduling work on them.

While it may be possible to use AZs for this, perhaps this pattern shouldn't appear in the RA document, as CNTT is largely organizing around exposing hardware and capabilities through NFVI profiles.
```suggestion
Availability Zones (AZs) rely on Host Aggregates and make the partitioning visible to tenants. They are defined by attaching specific metadata information to an aggregate, making the aggregate visible for tenants. Hosts can only be in a single Availability Zone. By default a host is part of a default Availability Zone, even if it doesn’t belong to an aggregate. Availability Zones can be used to provide resiliency and fault tolerance for workloads deployments, for example by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms. They permit rolling upgrades – an AZ at a time upgrade with enough time between AZ upgrades to allow recovery of tenant workloads on the upgraded AZ. 
```

Reply:
@kedmison That is fine. Thanks

@karinesevilla Up to you to incorporate the language. Thanks.